context-free.md

```diff
@@ -1,22 +1,40 @@
 # Context Free
 
 ##  Who is this programming language for?
-
+This programming language seems to be for a general audience, but using its more
+advanced features (the recursion aspects, for example) will require some
+knowledge of CS concepts. The overall grammar seems understandable, though it
```

Comment:
I agree. Writing code in ContextFree was still fairly challenging, and I don't know how I would have done it if I wasn't so experienced in programming already. This kind of goes against the idea in the reading that DSLs should make programming more accessible to subject matter experts because I think an artist would struggle using this if they had no prior programming experience.